diff --git a/node_modules/@credo-ts/core/build/modules/routing/services/MediationRecipientService.js b/node_modules/@credo-ts/core/build/modules/routing/services/MediationRecipientService.js
index 55a83f5..6b5c89d 100644
--- a/node_modules/@credo-ts/core/build/modules/routing/services/MediationRecipientService.js
+++ b/node_modules/@credo-ts/core/build/modules/routing/services/MediationRecipientService.js
@@ -97,7 +97,7 @@ let MediationRecipientService = class MediationRecipientService {
             },
         });
     }
-    async keylistUpdateAndAwait(agentContext, mediationRecord, updates, timeoutMs = 15000 // TODO: this should be a configurable value in agent config
+    async keylistUpdateAndAwait(agentContext, mediationRecord, updates, timeoutMs = 30000 // TODO: this should be a configurable value in agent config
     ) {
         var _a;
         const connection = await this.connectionService.getById(agentContext, mediationRecord.connectionId);
